curriculum/challenges/english/14-responsive-web-design-22/learn-typography-by-building-a-nutrition-label/615f887466db4ba14b5342cc.md

```diff
@@ -32,6 +32,12 @@ Your new `p` element should have the `class` attribute set to `no-divider`.
 assert(document.querySelector('.daily-value.small-text p:last-of-type')?.classList?.contains('no-divider'));
 ```
 
+You should have a `span` element with the class `bold` around the text `Protein`.
+
+```js
+assert(document.querySelector('.daily-value.small-text p:last-of-type span.bold')?.innerText.match(/Protein/));
```

Comment:
```suggestion
assert(document.querySelector('.daily-value.small-text p:last-of-type span.bold')?.innerText === 'Protein');
```

This passed for me: `<p class="no-divider"><span class="bold">Protein 3g</span></p>` (`3g` is in the `span`, I don't think that should pass). I tested the suggestion and it fixes it.

Reply:
Thank you for your review!